arcgis-ios-sdk-samples/Scenes/Surface placements/README.md

```diff
@@ -1,15 +1,42 @@
 # Surface placements
 
-This sample demonstrates how to position graphics using different surface placements
+Position graphics relative to a surface using different surface placement modes.
 
-![](image1.png)
+![Surface placements sample](surface-placements.png)
+
+## Use case
+
+Depending on the use case, data might be displayed at an absolute height (e.g. flight data recorded with altitude information), at a relative height to the terrain (e.g. transmission lines positioned relative to the ground), at a relative height to objects in the scene (e.g. extruded polygons, integrated mesh scene layer), or draped directly onto the terrain (e.g. location markers, area boundaries).
+
+## How to use the sample
+
+The sample loads a scene showing four points that use the individual surface placement rules (absolute, relative, and draped billboarded). Explore the scene by zooming in/out and by panning around to observe the effects of the surface placement rules.
 
 ## How it works
 
-The sample creates three graphic overlays. For each graphic overlay it set the `sceneProperties.surfacePlacement` property to one of the following:
+1. Create an `AGSGraphicsOverlay` for each placement mode, specifying the `surfacePlacement`: 
+    * `absolute`, position graphic using only its Z value.
+    * `relative`, position graphic using its Z value plus the elevation of the surface.
+    * `drapedBillboarded`, position graphic upright on the surface and always facing the camera, not using its z value.
+2. Add graphics to the graphics overlay using the `addObjects(from:)` method.
```

Comment:
I'm not sure the last part is needed since it is describing a `Foundation` API and there are various ways to update the `graphics` array:

```suggestion
2. Add graphics to the graphics overlay's `graphics` array.
```